src/components/app/data/services/subsidies/index.js

```diff
@@ -81,6 +104,16 @@ export async function fetchRedeemablePolicies(enterpriseUUID, userID) {
   }
 }
 
-export * from './browseAndRequest';
-export * from './subscriptions';
-export * from './couponCodes';
+// Policy Transaction
+
+/**
+ * Makes an API request to retrieve the most recent payload for the
+ * specified transaction. The transaction may be in various states such
+ * as pending, committed, etc.
+ * @param {Object} transaction
+ */
+export async function checkTransactionStatus(transaction) {
+  const { transactionStatusApiUrl } = transaction;
+  const response = await getAuthenticatedHttpClient().get(transactionStatusApiUrl);
+  return camelCaseObject(response.data);
+}
```

Comment:
In this particular case, I believe we want the error (if any) to throw. It's OK here because this call isn't made in the route loaders or on initial page load. The call is only triggered when clicking "Enroll" to redeem with Learner Credit; there is error handling around the stateful enroll logic that catches the error and displayed error messaging to the user, when applicable.